lang/english.xml

```diff
@@ -23,29 +23,29 @@
 		<Replace Tag="LOC_TRAIT_LEADER_ROOSEVELT_COROLLARY_ROUGH_RIDER_DESCRIPTION" Language="en_US">
-			<Text>Units receive +1 [ICON_Strength] Combat Strength for each reached individual era by technologies or civics (includes Ancient Era and up to Industrial Era, capped to +5). Envoys sent to City-states you have a [ICON_TradeRoute] Trade Route to count as two [ICON_Envoy] Envoys. Gain the Rough Rider unique unit with Ballistics technology.</Text>
+			<Text>Units receive +1 [ICON_Strength] Combat Strength for each individual player era reached by technologies or civics (includes Ancient Era and up to Industrial Era, capped to +5). Envoys sent to City-states you have a [ICON_TradeRoute] Trade Route to count as two [ICON_Envoy] Envoys. Gain the Rough Rider unique unit with Ballistics technology.</Text>
 		</Replace>
 		<Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_ANCIENT" Language="en_US">
-			<Text>+1 [ICON_Strength] Combat Strength for reached Ancient Era (Roosevelt Corollary).</Text>
+			<Text>+1 [ICON_Strength] Combat Strength for reaching Ancient Era (Roosevelt Corollary).</Text>
 		</Row>
 		<Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_CLASSICAL" Language="en_US">
-			<Text>+2 [ICON_Strength] Combat Strength for reached Classical Era (Roosevelt Corollary).</Text>
+			<Text>+2 [ICON_Strength] Combat Strength for reaching Classical Era (Roosevelt Corollary).</Text>
 		</Row>
 		<Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_MEDIEVAL" Language="en_US">
-			<Text>+3 [ICON_Strength] Combat Strength for reached Medieval Era (Roosevelt Corollary).</Text>
+			<Text>+3 [ICON_Strength] Combat Strength for reaching Medieval Era (Roosevelt Corollary).</Text>
 		</Row>
 		<Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_RENAISSANCE" Language="en_US">
-			<Text>+4 [ICON_Strength] Combat Strength for reached Renaissance Era (Roosevelt Corollary).</Text>
+			<Text>+4 [ICON_Strength] Combat Strength for reaching Renaissance Era (Roosevelt Corollary).</Text>
 		</Row>
 		<Row Tag="LOC_BBG_TEDDY_COMBAT_BONUS_IN_INDUSTRIAL" Language="en_US">
-			<Text>+5 [ICON_Strength] Combat Strength for reached Industrial Era (Roosevelt Corollary).</Text>
+			<Text>+5 [ICON_Strength] Combat Strength for reaching Industrial Era (Roosevelt Corollary).</Text>
 		</Row>
 		<!-- = leader ability (Abraham Lincoln)= -->
 		<Replace Tag="LOC_TRAIT_LEADER_LINCOLN_DESCRIPTION" Language="en_US">
-			<Text>Builds Industrial Zone district in half the time. Industrial Zone give +2 [ICON_Amenities] Amenities. Receive a free Melee unit after constructing Industrial Zone and their buildings. The free unit does not require resources when created or to maintain and receives +5 [ICON_Strength] Combat Strength.</Text>
+			<Text>+100% production toward . Industrial Zone give +2 [ICON_Amenities] Amenities. Receive a free Melee unit after constructing Industrial Zone and their buildings. The free unit does not require resources when created or to maintain and receives +5 [ICON_Strength] Combat Strength.</Text>
```

Comment:
Production (capitalized)
towards Industrial Zone District
?

Reply:
Production was not capitalized in Eleanor's ability, but probably should be in both here and on Eleanor.